poc/vdaf_prio3.py

```diff
@@ -86,15 +89,24 @@ def prep_init(Prio3, verify_key, agg_id, _agg_param,
             k_joint_rand_parts[agg_id] = k_joint_rand_part
             k_corrected_joint_rand = Prio3.joint_rand_seed(
                 k_joint_rand_parts)
-            joint_rand = Prio3.joint_rand(k_corrected_joint_rand)
+            joint_rands = Prio3.joint_rands(k_corrected_joint_rand)
 
         # Query the measurement and proof share.
-        query_rand = Prio3.query_rand(verify_key, nonce)
-        verifier_share = Prio3.Flp.query(meas_share,
-                                         proof_share,
-                                         query_rand,
-                                         joint_rand,
-                                         Prio3.SHARES)
+        query_rands = Prio3.query_rand(verify_key, nonce)
+        verifier_share = []
```

Comment:
For consistency with `shard()`
```suggestion
        verifiers_share = []
```

Reply:
Any chance you are not checking the [latest](i.e. 025c1c9) commit? Should be reflected, no?